cvmfs/download.h

```diff
@@ -138,6 +145,11 @@ struct JobInfo {
     probe_hosts = false;
     head_request = false;
     follow_redirects = false;
+    pid = -1;
```

Comment:
Another option is `CURLOPT_SSL_CTX_DATA` with the drawback that this binds curl to openssl.